index.d.ts

```diff
@@ -0,0 +1,37 @@
+declare module '@trex-arms/sqs' {
```

Comment:
I think the `declare module` syntax is for when you're typing an external module. https://stackoverflow.com/a/44060120/1509389

Reply:
I ran
```sh
npx -p typescript tsc index.mjs --declaration --allowJs --emitDeclarationOnly
```
and it produced this `index.d.mts` file:
```ts
declare function _default({ access_key_id, secret_access_key, region }: {
    access_key_id: any;
    secret_access_key: any;
    region: any;
}): {
    create_queue: (name: any, attributes?: {}) => Promise<any>;
    get_queue_url: (queue_name: any) => Promise<any>;
    get_queue_attributes: (queue_url: any, attribute_names?: any[]) => Promise<any>;
    delete_queue: (queue_url: any) => Promise<any>;
    send_message: (queue_url: any, message: any, { delay_seconds, message_attribute }?: {
        message_attribute?: {};
    }) => Promise<{
        message_id: any;
        md5_of_body: any;
    }>;
    send_message_batch: (queue_url: any, messages: any) => Promise<any>;
    receive_message: (queue_url: any, { max_number_of_messages, visibility_timeout, wait_time_seconds, attribute_names }?: {
        attribute_names?: any[];
    }) => Promise<any>;
    delete_message: (queue_url: any, receipt_handle: any) => Promise<any>;
};
export default _default;
```

I don't know the advantages/disadvantages of using `declare` vs `const sqs: CreateSqs`.  I'm guessing both work.

So I think if we get rid of line 1 and 37, we should be good to go.

Reply:
f2949edc6b1520e09478b1096a1c07f4c4b23e45